apps/wallet/src/ui/app/hooks/usePinnedCoinTypes.ts

```diff
@@ -1,51 +1,51 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { get, set } from 'idb-keyval';
-import { useCallback, useEffect, useState } from 'react';
+import { useCoinsStore } from '_app/zustand/coins';
+import { get } from 'idb-keyval';
+import { useCallback, useEffect } from 'react';
 
 import { useRecognizedPackages } from './useRecognizedPackages';
 
 const PINNED_COIN_TYPES = 'pinned-coin-types';
 
 export function usePinnedCoinTypes() {
+	const setPinnedCoinTypes = useCoinsStore((state) => state.setPinnedCoinTypes);
```

Comment:
You don't really need selectors here at all since you pull the entire store through this hook + the next one, probablyjust use `useCoinsStore` with no selector and destructure.

Reply:
Yep, fixed that